<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QMainWindow" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nelia</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabwidget_main">
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab_project">
       <attribute name="title">
        <string>Project</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_log">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Log</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="margin">
         <number>0</number>
        </property>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_roadmap">
       <attribute name="title">
        <string>Roadmap</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6"/>
      </widget>
      <widget class="QWidget" name="tab_repository">
       <attribute name="title">
        <string>Repository</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>470</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <action name="action_open_db">
   <property name="text">
    <string>&amp;Open..</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_save_db">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_save_as_db">
   <property name="text">
    <string>Save &amp;As..</string>
   </property>
  </action>
  <action name="action_preferences">
   <property name="text">
    <string>&amp;Preferences..</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_merge_db">
   <property name="text">
    <string>Merge..</string>
   </property>
  </action>
  <action name="action_new_db">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="action_new_entry">
   <property name="text">
    <string>&amp;New..</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_change_entry">
   <property name="text">
    <string>&amp;Change..</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_delete_entry">
   <property name="text">
    <string>&amp;Delete</string>
   </property>
  </action>
  <action name="action_path">
   <property name="text">
    <string>Path..</string>
   </property>
  </action>
  <action name="action_log">
   <property name="text">
    <string>Log..</string>
   </property>
  </action>
  <action name="action_map">
   <property name="text">
    <string>Map..</string>
   </property>
  </action>
  <action name="action_data">
   <property name="text">
    <string>Data..</string>
   </property>
  </action>
  <action name="action_protocol">
   <property name="text">
    <string>Protocol..</string>
   </property>
  </action>
  <action name="action_blueprint">
   <property name="text">
    <string>Concepts..</string>
   </property>
  </action>
  <action name="action_roadmap">
   <property name="text">
    <string>Roadmap..</string>
   </property>
  </action>
  <action name="action_issues">
   <property name="text">
    <string>Issues..</string>
   </property>
  </action>
  <action name="action_features">
   <property name="text">
    <string>Features..</string>
   </property>
  </action>
  <action name="action_refactor">
   <property name="text">
    <string>Refactor..</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>&amp;About..</string>
   </property>
  </action>
  <action name="action_manual">
   <property name="text">
    <string>&amp;Manual..</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tabwidget_main</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>action_quit</sender>
   <signal>triggered()</signal>
   <receiver>mainwindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
